lib/Graph/Graph.cpp

```diff
@@ -5317,4 +5317,27 @@ llvm::raw_ostream &operator<<(llvm::raw_ostream &os, const Function *F) {
   F->dump(os);
   return os;
 }
+
+bool isConvolutionSameAsFullyConnected(const ConvolutionNode *node) {
+  bool isConv2D = (node->getInput().getType()->dims().size() == 4);
+  if (!(isConv2D && node->getLayout() == ConvolutionLayout::NHWC)) {
```

Comment:
We should also check that this conv doesn't have a fused activation:
!node->hasFusedActivation()

Reply:
Good point. I will add this (and below) in a minute.

Reply:
Done.